rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -0,0 +1,84 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_maintain
+%global confdir foreman_maintain
+
+%{!?_root_bindir:%global _root_bindir %{_bindir}}
+
+Summary: The Foreman/Satellite maintenance tool
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?dist}
+Group: Development/Languages
+License: GPLv3
+URL: http://github.com/theforeman/foreman_maintain
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(clamp) >= 0.6.2
+Requires: %{?scl_prefix}rubygem(highline)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%description
+foreman_maintain aims to provide various features that helps keeping
+the Foreman/Satellite up and running. It supports multiple versions
```

Comment:
This string too is taken from [foreman_maintain upstream](https://github.com/iNecas/foreman_maintain)